express/scripts/scripts.js

```diff
@@ -1000,6 +1001,53 @@ function addPromotion() {
   }
 }
 
+/**
+ *  Load Adobe Clean Han font family for supported locale pages.
+ *  Typekit IDs of each locale are consistent with corresponding home pages on adobe.com.
+ */
+function loadAdobeCleanHanTypekit() {
+  const KIT_IDS = {
+    jp: 'dvg6awq',
+    kr: 'qjs5sfm',
+    cn: 'puu3xkp',
+    tw: 'jay0ecd',
+  };
+
+  function loadTypekit(d, locale) {
+    const config = {
+      kitId: KIT_IDS[locale],
+      scriptTimeout: 3000,
+      async: true,
+    };
+    const h = d.documentElement;
+    const t = setTimeout(() => {
+      h.className = `${h.className.replace(/\bwf-loading\b/g, '')} wf-inactive`;
+    }, config.scriptTimeout);
+    let f = false;
+    let a;
+    h.className += ' wf-loading';
```

Comment:
I prefer using the `classList` object as a cleaner and safer way to manipulate an element's classes.